gen3/mkdocs.yml

```diff
@@ -122,8 +125,9 @@ plugins:
       blog_toc: true
   - mkdocs-video:
       is_video: True
-extra_css:
-  - stylesheets/extra.css
+  - encryptcontent:
+      password_inventory:
+        _global: 'gen3'
```

Comment:
I realize this isn't necessarily intended to be secure, but having this plain text sorta defeats the purpose of trying to hide the content on the branches. I think this needs to be a real Github secret

Reply:
This is just so that random people don't run into it for now, until we're ready to publish.

Reply:
Yeah but the repo is public so anyone can see this. So long as it's well understood that this isn't actually protecting against anything and folks writing the documentation understand that this does not actually _prevent_ anyone from seeing the content, then it's okay. But configuring a secret on the repo and using that as a var here isn't too complicated and would alleviate my concern entirely

Reply:
Yeah, the intention behind this password is for crawlers not to index this page, not to hide the content from the public.